### PR TITLE
Removed no from creating runtime app

### PR DESCRIPTION
**Pull request by jonaswre**
Removed the number from runtime app as talked about in #1741

### CODE REVIEW

**Comment by jonaswre** [comment]
@freddydk can you review this change?

**Comment by freddydk** [comment]
Will do - I will have to access the consequences of this change before doing it.

File: ArtifactHandling/Resolve-DependenciesFromAzureFeed.ps1 [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    Resolve-DependenciesFromAzureFeed -organization $organization -feed $feed -outPath $outPath -appsPath $tempAppDependencyFolder -spaces $spaces
+                } else {
+                    Write-Host "$($spaces)$($dep.Name) exists"
+                }
+            } 
+        }
+        if ($appJson.dependencies.Count -eq 0) {
+            Write-Host "$($spaces)No more dependencies."
+        }
+        Remove-Item -Path $tempAppFolder -Recurse -Force
```
Comment (freddydk) [inline ArtifactHandling/Resolve-DependenciesFromAzureFeed.ps1:113]: This should be in a finally section

File: ArtifactHandling/Resolve-DependenciesFromAzureFeed.ps1 [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            Write-Host "$($spaces)$($_.DirectoryName | Split-Path -Leaf)"
+        } else {
+            Write-Host "$($spaces)$($_.Name)"
+        }
+
+        # Create temp folders
+        $tempAppFolder = Join-Path ((Get-Item -Path $env:temp).FullName) ([Guid]::NewGuid().ToString())
+        $tempAppDependenciesFolder = Join-Path $tempAppFolder 'dependencies'
+
+        New-Item -path $tempAppFolder -ItemType Directory -Force | Out-Null
```
Comment (freddydk) [inline ArtifactHandling/Resolve-DependenciesFromAzureFeed.ps1:58]: You should add a try/finally when creating a temp folder - to clean up is something fails

File: ArtifactHandling/Publish-BuildOutputToAzureFeed.ps1 [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        [Parameter(Mandatory=$true)]
+        [string] $path,
+        [Parameter(Mandatory=$true)]
+        [string] $pat
+    )
+    Get-Childitem –Path (Join-Path $path "\Apps\*.app") | % {
+        $basename = $_.Basename
+
+        $tempAppFolder = Join-Path ((Get-Item -Path $env:temp).FullName) ([Guid]::NewGuid().ToString())
+        $tempAppOutFolder = Join-Path $tempAppFolder "out"
```
Comment (freddydk) [inline ArtifactHandling/Publish-BuildOutputToAzureFeed.ps1:35]: Add a try/finally to ensure that the removal of the temp folder is executed

File: ArtifactHandling/Publish-BuildOutputToAzureFeed.ps1 [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+        az artifacts universal publish `
+            --organization $organization `
+            --feed $feed `
+            --name $appJson.id `
+            --version $appVersion `
+            --description $appJson.name `
+            --path (Join-Path $tempAppOutFolder '.') `
+
+        Remove-Item -Path $tempAppFolder -Recurse -Force
```
Comment (freddydk) [inline ArtifactHandling/Publish-BuildOutputToAzureFeed.ps1:72]: This in the finally

**Comment by freddydk** [comment]
Very nice, I have been thinking about Az Artifacts support, but never got to look into this.
I have briefly scanned the code and it looks fine.
I think you need to sign the contributor license in order for me to merge the PR.

**Comment by jonaswre** [comment]
@freddydk  I've created a new PR for that only azure feed support. #1769

**Pull request closed without merging**